adapta/security/clients/_local_client.py

```diff
@@ -56,3 +56,9 @@ def connect_account(self):
 
     def get_pyarrow_filesystem(self, path: DataPath, connection_options: Optional[Dict[str, str]] = None) -> FileSystem:
         return SubTreeFileSystem(path.path, LocalFileSystem())
+
+    def initialize_session(self, session_callable=None) -> "LocalClient":
```

Comment:
Type hints

Reply:
Missing indeed!
Added in [3943e2d](https://github.com/SneaksAndData/adapta/pull/437/commits/3943e2dda0291afba169f072092af8a0ff6741b7)
Thank you for pointing it out 🙂